connection.js

```diff
@@ -985,6 +986,7 @@ Connection.prototype.rcpt_ok_respond = function (retval, msg) {
         this.results.add(this, {err: 'rcpt_ok_respond found no transaction'});
         return;
     }
+    if (!msg) msg = this.last_rcpt_msg;
     var rcpt = this.transaction.rcpt_to[this.transaction.rcpt_to.length - 1];
```

Comment:
curiosity question:
Couldn't the first change, setting `last_rcpt_msg=null` be omitted? 

And likewise, wouldn't this be better?

```diff
- if (!msg) msg = this.last_rcpt_msg;
+ if (!msg && this.last_rcpt_msg) msg = this.last_rcpt_msg;

- this.last_rcpt_msg = msg;
+ if (msg) this.last_rcpt_msg = msg;
```